pkg/minikube/reason/reason.go

```diff
@@ -64,54 +64,98 @@ var (
-	InternalReservedProfile  = Kind{ID: "MK_RESERVED_PROFILE", ExitCode: ExProgramConflict}
-	InternalEnvScript        = Kind{ID: "MK_ENV_SCRIPT", ExitCode: ExProgramError}
-	InternalShellDetect      = Kind{ID: "MK_SHELL_DETECT", ExitCode: ExProgramError}
-	InternalStatusJSON       = Kind{ID: "MK_STATUS_JSON", ExitCode: ExProgramError}
-	InternalStatusText       = Kind{ID: "MK_STATUS_TEXT", ExitCode: ExProgramError}
-	InternalUnsetScript      = Kind{ID: "MK_UNSET_SCRIPT", ExitCode: ExProgramError}
-	InternalViewExec         = Kind{ID: "MK_VIEW_EXEC", ExitCode: ExProgramError}
-	InternalViewTmpl         = Kind{ID: "MK_VIEW_TMPL", ExitCode: ExProgramError}
-	InternalYamlMarshal      = Kind{ID: "MK_YAML_MARSHAL", ExitCode: ExProgramError}
-	InternalCredsNotFound    = Kind{ID: "MK_CREDENTIALS_NOT_FOUND", ExitCode: ExProgramNotFound, Style: style.Shrug}
-	InternalCredsNotNeeded   = Kind{ID: "MK_CREDENTIALS_NOT_NEEDED", ExitCode: ExProgramNotFound, Style: style.Shrug}
-	InternalSemverParse      = Kind{ID: "MK_SEMVER_PARSE", ExitCode: ExProgramError}
-	DaemonizeError           = Kind{ID: "MK_DAEMONIZE", ExitCode: ExProgramError}
+	// minikube failed to list some configuration data
+	InternalListConfig = Kind{ID: "MK_LIST_CONFIG", ExitCode: ExProgramError}
+	// minikube failed to write logs to stdout (currently not in use)
+	InternalLogtostderrFlag = Kind{ID: "MK_LOGTOSTDERR_FLAG", ExitCode: ExProgramError}
+	// minikube failed to follow or watch minikube logs
+	InternalLogFollow = Kind{ID: "MK_LOG_FOLLOW", ExitCode: ExProgramError}
+	// minikube failed to create an appropriate new runtime based on the driver in use
+	InternalNewRuntime = Kind{ID: "MK_NEW_RUNTIME", ExitCode: ExProgramError}
+	// minikube was passed an invalid value for the --output command line flag
+	InternalOutputUsage = Kind{ID: "MK_OUTPUT_USAGE", ExitCode: ExProgramError}
+	// minikube could not configure the runtime in use, or the runtime failed
+	InternalRuntime = Kind{ID: "MK_RUNTIME", ExitCode: ExProgramError}
+	// minikube was passed a reserved keyword as a profile name, which is not allowed
+	InternalReservedProfile = Kind{ID: "MK_RESERVED_PROFILE", ExitCode: ExProgramConflict}
+	// minkube failed to generate script to set or unset minikube-env
+	InternalEnvScript = Kind{ID: "MK_ENV_SCRIPT", ExitCode: ExProgramError}
+	// minikube failed to detect the shell in use
+	InternalShellDetect = Kind{ID: "MK_SHELL_DETECT", ExitCode: ExProgramError}
+	// minikube failed to output JSON-formatted minikube status
+	InternalStatusJSON = Kind{ID: "MK_STATUS_JSON", ExitCode: ExProgramError}
+	// minikube failed to output minikube status text
+	InternalStatusText = Kind{ID: "MK_STATUS_TEXT", ExitCode: ExProgramError}
+	// minikube failed to generate script to deactive minikube docker-env
+	InternalUnsetScript = Kind{ID: "MK_UNSET_SCRIPT", ExitCode: ExProgramError}
+	// minikube failed to create view template for displaying current config
```

Comment:
Flip this comment with the one on the next line. InternalViewExec is about executing the view template, and InternalViewTmpl is about creating the view template.